runtime/queries/toml/highlights.scm

```diff
@@ -7,6 +7,12 @@
   (quoted_key)
 ] @type)
 
+(table [
+  (bare_key)
+  (dotted_key)
+  (quoted_key)
+] @key.declaration)
+
```

Comment:
there are theme key changes and even an unrelated (and incorrect) code change (which will cause build failures) bundled with this PR. A PR adding a theme should not contain such unrelated changes, please remove them